Makefile

```diff
@@ -1,77 +1,7 @@
-.PHONY: all install clean doc github test cdtest cdtest_tls
-
-EXT_OBJ:=$(shell ocamlfind ocamlc -config | awk '/^ext_obj:/ {print $$2}')
-EXT_LIB:=$(shell ocamlfind ocamlc -config | awk '/^ext_lib:/ {print $$2}')
-
-OCAMLBUILD = ocamlbuild -use-ocamlfind -classic-display \
-	-cflags "-w A-4-33-40-41-42-43-34-44" \
-	-plugin-tag "package(ppx_driver.ocamlbuild)"
-
-PREFIX ?= /usr/local/bin
-OS_TYPE:=$(shell ocamlfind ocamlc -config | awk '/^os_type:/ {print $$2}')
-ifeq ($(OS_TYPE),$(filter $(OS_TYPE),Win32 Cygwin))
-EXT_EXE=.exe
-else
-EXT_EXE=
-endif
-
-
-B=_build/lib
-FILES = $(wildcard $B/*.cmi $B/*.cmt $B/*.cmti $B/*.cmx $B/*.cmxa $B/*.cma $B/*.cmxs $B/*$(EXT_LIB) $B/*$(EXT_OBJ) $B/*.cmo)
-MORE_FILES = $(wildcard lib/intro.html $B/*.mli)
+.PHONY: all clean
 
 all:
-	$(OCAMLBUILD) conduit.otarget
-
-install:
-	rm -rf _install
-	mkdir -p _install
-ifneq ("$(wildcard _build/lib/conduit_xenstore.cmo)","")
-	echo '"scripts/xenstore-conduit-init" {"xenstore-conduit-init"}' > _install/bin
-endif
-	$(foreach f,$(FILES), echo "$(f)" >> _install/lib;)
-	ocamlfind remove conduit || true
-	ocamlfind install conduit META $(FILES) $(MORE_FILES)
+	jbuilder build
```

Comment:
no that breaks the build as there are known warnings. will add in a future release